client/v2/autocli/flag/field.go

```diff
@@ -60,8 +61,14 @@ func (b *Builder) addFieldFlag(ctx context.Context, flagSet *pflag.FlagSet, fiel
 
 	// use the built-in pflag StringP, Int32P, etc. functions
 	var val HasValue
+
 	if field.IsList() {
 		val = bindSimpleListFlag(flagSet, field.Kind(), name, shorthand, usage)
+	}
+	if field.IsMap() {
```

Comment:
Should be else if, otherwise the list will not be supported anymore, as the else below will be true.